package.json

```diff
@@ -49,8 +49,7 @@
   "scripts": {
     "test": "lab -a code -t 100 -L -m 3000",
     "test-tap": "lab -a code -r tap -o tests.tap -m 3000",
-    "test-cov-html": "lab -a code -r html -o coverage.html -m 3000",
-    "postshrinkwrap": "json -I -f npm-shrinkwrap.json -e 'f(this) ; function f (p) { var deps = p.dependencies || {}; Object.keys(deps).forEach(function (n) { delete deps[n].resolved; f(deps[n]) }) }'"
```

Comment:
I did! Because the final commit makes the `npm-shrinkwrap.json` be entirely unedited, and thus the `postshrinkwrap` script no longer has any purpose. The idea here is to illustrate some options on what one might do with the shrinkwrap over the course of the commits. One can crop this commit log anywhere after devdeps are added and it'll produce the kind of shrinkwrap in that commit.